src/UIComponents/Statement.jsx

```diff
@@ -29,9 +31,10 @@ class Statement extends Component {
   };
 
   labelHTML = (label , currentPanel) => {
+    const localizedLabel = getLocalizedColumnName(this.props.datasetId, label);
     return (
       <div style={styles.statementLabel}>
-        {label || "____"}
+        {(label && localizedLabel) || "____"}
```

Comment:
Slightly confused by the order here. We would want to either render `localizedLabel` if it exists over rendering the given `label` and want `localizedLabel && label` ... but at any rate, localizedLabel will always either have a localized string OR fallback to `label` anyway?

Reply:
yeah you are right we can just use localizedLabel instead of label.